ner_v2/detectors/temporal/constant.py

```diff
@@ -0,0 +1,24 @@
+DATE_CONSTANT_FILE = 'date_constant.csv'
+TIME_CONSTANT_FILE = 'time_constant.csv'
+DATETIME_CONSTANT_FILE = 'datetime_diff_constant.csv'
+NUMERALS_CONSTANT_FILE = 'numbers_constant.csv'
+
+CONSTANT_FILE_KEY = 'key'
+
+MONTH_DATE_REF_TYPE = 'month_date_ref'
+RELATIVE_DATE = 'relative_date'
+DATE_LITERAL_TYPE = 'date_literal'
+MONTH_LITERAL_TYPE = 'month_literal'
+WEEKDAY_TYPE = 'weekday'
+MONTH_TYPE = 'month'
+ADD_DIFF_DATETIME_TYPE = 'add_diff_datetime'
+REF_DATETIME_TYPE = 'ref_datetime'
```

Comment:
Put inline comments for constants from lines 8 to 15.

Reply:
done